examples/alice/css/core.css

```diff
@@ -1,75 +1,69 @@
-@charset "UTF-8";
```

Comment:
Any styling changes _within the book content_ have to be done in js, the way I have it set up now. For the nav and any of the book controls, I think we should include a new css file from index.html.